setup.py

```diff
@@ -54,7 +54,7 @@ def package_data(pkg, root_list):
     'pb-choice = problem_builder:ChoiceBlock',
 
     'pb-dashboard = problem_builder:DashboardBlock',
-    'pb-data-export = problem_builder:DataExportBlock',
```

Comment:
Can you please leave `pb-dashboard` in with a comment saying `deprecated`, so that existing courses will keep working?